ops/scripts/update-op-geth.py

```diff
@@ -10,7 +10,7 @@
 
 
 def main():
-	for project in ('.', 'indexer'):
+	for project in ('.',):
```

Comment:
```suggestion
	for project in ('.'):
```
or just remove the loop completely and just call next 2 lines with `.`.

Reply:
actually, we can just remove the whole `update-op-geth` script.

Reply:
> actually, we can just remove the whole `update-op-geth` script.

want to follow up with this @sebastianst ?